pkg/appmanager/as3Manager.go

```diff
@@ -480,12 +480,18 @@ func (as3RestClient *AS3RESTClient) restCallToBigIP(method string, route string,
 		}
 		as3RestClient.oldChecksum = as3RestClient.newChecksum
 		return string(body), true
+	}
+	//Other then 200 status code
+	if results, ok := (response["results"]).([]interface{}); ok {
+		for _, value := range results {
+			v := value.(map[string]interface{})
+			//log result with code, tenant and message
+			log.Debugf("[as3_log] Big-IP Response error code: %v,response:%v, message: %v", v["code"], v["response"], v["message"])
```

Comment:
We might get valid messages other than 200 also.
```suggestion
			log.Debugf("[as3_log] Big-IP Response code: %v, Response:%v, Message: %v", v["code"], v["response"], v["message"])
```